science/libmed/Portfile

```diff
@@ -3,10 +3,14 @@
 PortSystem              1.0
 PortGroup               mpi       1.0
 PortGroup               cmake     1.1
+PortGroup               legacysupport 1.1
+
+# MEDfileExist.c: error: implicit declaration of function 'strndup' [-Wimplicit-function-declaration]
+legacysupport.newest_darwin_requires_legacy 10
 
 name                    libmed
 version                 4.1.1
-revision                3
+revision                2
```

Comment:
why are you decreasing the `revision`, I'm assuming that is a mistake?

Reply:
Yes, of course. Thank you, will fix now.